### PR TITLE
Bug 1793144: kubelet: add log level environment variable

### PR DESCRIPTION
**Pull request by rphillips**
**- What I did**
Create an environment variable for the kubelet log level, so it can be overridden in /etc/kubernetes/kubelet-env across reboots.

**- How to verify it**

**- Description for the changelog**
```NONE```

### CODE REVIEW

File: templates/master/01-master-kubelet/_base/units/kubelet.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ contents: |
   Type=notify
   ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
   ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
+  Environment="KUBELET_LOG_LEVEL=3"
```
Comment (mrunalp) [inline templates/master/01-master-kubelet/_base/units/kubelet.yaml]: Did we test that we can override this in one of the environment files?

Comment (rphillips) [reply templates/master/01-master-kubelet/_base/units/kubelet.yaml]: yes. it works fine.

**Comment by umohnani8** [comment]
LGTM

**Comment by umohnani8** [comment]
Maybe add an example in the doc somewhere on how to use it?

**Comment by cgwalters** [comment]
xref https://github.com/openshift/machine-config-operator/pull/1439

I think what we *really* want here is `/etc/kubelet.conf.d`.

**Pull request merged**